npm/npm_test.go

```diff
@@ -46,10 +46,11 @@ func newNPMgr(t *testing.T, exec utilexec.Interface) *NetworkPolicyManager {
 
 func TestMain(m *testing.M) {
 	metrics.InitializeAll()
-	iptMgr := iptm.NewIptablesManager()
```

Comment:
Do we still need iptMgr and ipsMgr and methods of them in npm_test.go since we do not use any real management of iptables and ipset in our UTs with this PR? I assume we do not need them here anymore. If so, it is safe to remove them to avoid uncertain situation in test environments from managements of real iptables and ipset.